visual-js/visual-snapshots/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@saucelabs/visual-snapshots",
+  "description": "CLI which generates Visual snapshots from a data source such as pdf",
+  "version": "0.1.0",
+  "main": "lib/index.cjs",
+  "module": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "license": "MIT",
+  "bin": "./lib/index.js",
+  "files": [
+    "build",
+    "README.md"
+  ],
+  "type": "module",
+  "engines": {
+    "node": "^16.13 || >=18"
+  },
+  "keywords": [
+    "saucelabs",
+    "visual",
+    "snapshots",
+    "pdf"
+  ],
+  "exports": {
+    ".": {
+      "require": "./lib/index.cjs",
+      "import": "./lib/index.js",
+      "types": "./lib/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
```

Comment:
I think this is redundant and we can remove this. 
This is useful for multiple entry points, which we do not use in this app.

https://docs.npmjs.com/cli/v11/configuring-npm/package-json#exports